.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
```

Comment:
We could alternatively add this config and the black line-length to the code template. I'm not sure that makes sense though, ultimately the way the code is styled feels like something we might want to maintain separately.